ocaml-backend/src/main/java/org/kframework/backend/ocaml/DefinitionToOcaml.java

```diff
@@ -398,16 +398,19 @@ public static String enquoteString(byte[] value) {
 
     public String execute(K k, int depth, String file) {
         StringBuilder sb = new StringBuilder();
-        ocamlProgramHeader(sb, true);
+        ocamlProgramHeader(sb, true, "open List\n");
```

Comment:
I don't like this; List has a bunch of functions in it that might not behave well when `open`ed. You should just prefix the functions with List.

Reply:
Ok.